iozone/iozone_run.sh

```
@@ -680,8 +680,8 @@ do_test_actual()
 			else
 				let "file_size=$max_file_size"
 				echo ================================================ >> ${iozone_output_file}
-				echo ${iozone_exe} -R ${4} ${6} -r ${page_size} -s${file_size}g -c -w -C ${iozone_args}-F ${5} >> ${iozone_output_file}
-				time taskset -c 0 ${iozone_exe} -R ${4} ${6} -r ${page_size} -s${file_size}g -c -w -C ${iozone_args} -F ${5} >> ${iozone_output_file}
+				echo ${iozone_exe} -R ${4} ${6} -r ${min_file_size} -s${file_size}g -c -w -C ${iozone_args}-F ${5} >> ${iozone_output_file}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)